app/views/directives/ui-ace-yaml.html

```diff
@@ -0,0 +1,44 @@
+<ng-form name="$ctrl.form">
+  <div class="form-group" id="yaml-file">
+    <osc-file-input
+      ng-if="$ctrl.showFileInput"
+      model="$ctrl.fileUpload"
+      drop-zone-id="yaml-file"
+      help-text="Upload file by dragging & dropping, selecting it, or pasting from the clipboard."
+      ng-disabled="false"
+      hide-clear="true"></osc-file-input>
+
+    <div class="edit-yaml-errors">
+      <!--
+        Since the YAML parser quits after the first error, don't give a count.
+        Just show an indicator and link to the first problem line.
+      -->
+      <div ng-if="firstError = $ctrl.annotations.error[0]">
```

Comment:
I think it's subtle enough? Opinion? It might be stranger for it to appear and disappear when you scroll

![openshift web console 2017-07-11 15-15-00](https://user-images.githubusercontent.com/1167259/28086075-bef14470-664b-11e7-9191-de4fc2ae96f8.png)